code/modules/mob/new_player/new_player.dm

```diff
@@ -508,10 +508,10 @@
 		ticker.minds += character.mind//Cyborgs and AIs handle this in the transform proc.	//TODO!!!!! ~Carn
 
 		//Grab some data from the character prefs for use in random news procs.
-
-		AnnounceArrival(character, rank, join_message)
-	else
-		AnnounceCyborg(character, rank, join_message)
+		if(!character.mind.prefs.silent_join)
+			AnnounceArrival(character, rank, join_message)
+		else
```

Comment:
This indentation looks kinda sus.

Specifically that it looks like people with silent join on will be announced as if they were borgs.

Reply:
Fixed it! Good catch, didn't realize during testing